contracts/curve/types/JumpRateUtilizationCurve.sol

```diff
@@ -48,6 +48,19 @@ library JumpRateUtilizationCurveLib {
         }
         return self.maxRate.unpack();
     }
+
```

Comment:
can we name these `JumpRateUtilizationCurve18` for clarity? (will require some changes to upgrade to this, but should be simple)